tools/cloud-build/daily-tests/blueprints/lustre-slurm-v6.yaml

```diff
@@ -0,0 +1,149 @@
+  rocky_image:
+    family: slurm-gcp-6-4-hpc-rocky-linux-8
+    project: schedmd-slurm-public
+
+deployment_groups:
+- group: primary
+  modules:
+
+  - id: network
+    source: modules/network/pre-existing-vpc
+
+  ###########
+  # Storage #
+  ###########
+
+  # This file system has an associated license cost.
+  # https://console.developers.google.com/marketplace/product/ddnstorage/exascaler-cloud
+  - id: lustre
+    source: community/modules/file-system/DDN-EXAScaler
+    use: [network]
+    settings:
+      local_mount: /lustre
+      waiter: deploymentmanager
+      mgs:
+        nic_type: "GVNIC"
+        node_type: n2-standard-2
+        node_count: 1
+        node_cpu: "Intel Cascade Lake"
+        public_ip: true
+      mds:
+        nic_type: "GVNIC"
+        node_type: n2-standard-2
+        node_count: 1
+        node_cpu: "Intel Cascade Lake"
+        public_ip: true
+      oss:
+        nic_type: "GVNIC"
+        node_type: n2-standard-2
+        node_count: 3
+        node_cpu: "Intel Cascade Lake"
+        public_ip: true
+
+  #############
+  # Slurm VMs #
+  #############
+
+  # # Ubuntu 20.04 LTS
+  # - id: ubuntu_nodeset
+  #   source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+  #   use: [network]
+  #   settings:
+  #     node_count_dynamic_max: $(vars.num_nodes)
+  #     instance_image:
+  #       family: slurm-gcp-6-4-ubuntu-2004-lts
+  #       project: schedmd-slurm-public
+
+  # - id: ubuntu_partition
+  #   source: community/modules/compute/schedmd-slurm-gcp-v6-partition
+  #   use:
+  #   - ubuntu_nodeset
+  #   - lustre
+  #   settings:
+  #     partition_name: ubuntu
+
+  # Rocky Linux 8
+  - id: rocky_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network]
+    settings:
+      node_count_dynamic_max: $(vars.num_nodes)
+      instance_image: $(vars.rocky_image)
+
+  - id: rocky_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v6-partition
+    use:
+    - rocky_nodeset
+    settings:
+      partition_name: rocky
+
+  # CentOS 7
+  - id: centos_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network]
+    settings:
+      node_count_dynamic_max: $(vars.num_nodes)
+      instance_image: $(vars.centos_image)
+
+  - id: centos_partition
+    source: community/modules/compute/schedmd-slurm-gcp-v6-partition
+    use:
+    - centos_nodeset
+    settings:
+      partition_name: centos
+
+  - id: slurm_login
+    source: community/modules/scheduler/schedmd-slurm-gcp-v6-login
+    use: [network]
+    settings:
+      name_prefix: login
+      disable_login_public_ips: false
+
+  - id: slurm_controller
+    source: community/modules/scheduler/schedmd-slurm-gcp-v6-controller
+    use:
+    - network
+    # - ubuntu_partition
+    - rocky_partition
+    - centos_partition
+    - lustre
+    - slurm_login
+    settings:
+      disable_controller_public_ips: false
+      # cloud_parameters:
```

Comment:
Perhaps remove all the commented lines? Why are they here?

Reply:
There are coming from slurm-gcp v5 version. I removed these commended parameters for controller. For ubuntu compute nodes, it seems in v5 we had 3 os and removed ubuntu for some reason.

Reply:
lmkwyt.